src/lio/lio_core_os.c

```diff
@@ -579,12 +580,118 @@ gop_op_generic_t *gop_lio_abort_regex_object_set_multiple_attrs(lio_config_t *lc
 }
 
 //*************************************************************************
-// lio_move_op - Renames an object
+//  lio_is_dir_empty - Returns 0 if the directory is empty and a non-0 otherwise
```

Comment:
I made the change and also munged all the gop_lio_* -> lio_*_gop like we 
discussed yesterday on slack.  I also found a bunch of others using a 
lio_*_op or just lio_* naming conventions that all returned GOPs and 
normalized those as well.

BTW you're going to hate the feature/tests merge.  I've been sprinkling 
in all the changes on the PR thread in with the test changes.  At least 
they are all separate commits for cherry picking if needed.